lib/DXIL/DxilShaderModel.cpp

```diff
@@ -494,6 +494,18 @@ const llvm::StringRef ShaderModel::FullNameFromKind(DXIL::ShaderKind sk) {
   }
 }
 
+bool ShaderModel::AllowDerivatives(DXIL::ShaderKind sk) {
```

Comment:
I think this function needs shader model as well, since derivatives were enabled in compute+amp+mesh in SM 6.6.

Reply:
Fixed.